django/contrib/sitemaps/views.py

```diff
@@ -10,6 +10,15 @@
 from django.utils.http import http_date
 
 
+class SiteData:
+    def __init__(self, location, last_mod=None):
+        self.location = location
+        self.last_mod = last_mod
+
+    def __str__(self):
+        return self.location
```

Comment:
We should probably keep this class. I think I'd call it `SitemapIndexItem`, and give it a docstring. 🤔 

I guess deprecating the `__str__()` usage will keep everyone on the same page. 

(Without something like this, we're still passing a list of pair or dicts or ... — given that we have the class already, we may as well keep it.)

Reply:
I've changed this to a `dataclass`, what do you think about this? I don't think Django has any as yet? 

Following this change I'm stuggling for words on a doc string, well one that doesn't just say it's a dataclass to hold location and last_mod. What were you thinking of adding?

Reply:
I think it's just right as a dataclass 👍